integrationTests/json/mandosAdderLog_test.go

```diff
@@ -20,8 +20,8 @@ VM hook begin: GetNumArguments() points used: 110
 VM hook end:   GetNumArguments() points used: 210
 VM hook begin: BigIntGetUnsignedArgument(0, -101) points used: 249
 VM hook end:   BigIntGetUnsignedArgument(0, -101) points used: 1249
-VM hook begin: MBufferSetBytes(-102, 1048601, 3) points used: 1289
-VM hook end:   MBufferSetBytes(-102, 1048601, 3) points used: 6289
+VM hook begin: MBufferSetBytes(-102, 1048576, 3) points used: 1289
```

Comment:
are these changes ok?

Reply:
Yes, the Rust->Wasm compiler changed the order of two constant byte slices, for some reason, hence the second argument changed, which is a pointer. This is the infrastructure to compare wasm1 to wasm2.